src/main/scala/akka/contrib/stream/InputStreamPublisher.scala

```diff
@@ -16,6 +16,7 @@ import scala.concurrent.duration.Duration
 /**
  * Companion for [[InputStreamPublisher]] – defines a Props factory.
  */
+@deprecated(message = "Removed due to Akka stream support for creating Source and Sink from InputStream and OutputStream", since = "2.0")
```

Comment:
We should just remove these from here - let's put the deprecation messages against the classes on the 1.0 branch.